packages/devtools_app/lib/src/screens/memory/memory_controller.dart

```diff
@@ -140,9 +76,9 @@ class MemoryController extends DisposableController
   final _shouldShowLeaksTab = ValueNotifier<bool>(false);
   ValueListenable<bool> get shouldShowLeaksTab => _shouldShowLeaksTab;
 
-  ValueListenable get legendVisibleNotifier => _legendVisibleNotifier;
+  ValueListenable<bool> get legendVisibleNotifier => _legendVisibleNotifier;
 
-  final _legendVisibleNotifier = ValueNotifier<bool>(false);
+  final _legendVisibleNotifier = ValueNotifier<bool>(true);
```

Comment:
should true really be the default. That will make the UI very busy IMO

Reply:
requested by Mindy, because of some UXR

@melyndahoover , can you add details?

Reply:
This was a suggestion based on an informal cognitive walk through. Essentially, it is not very discoverable if it is hidden by default. But perhaps there is a compromise. From the few examples I have seen of people using this tool, it seems that the majority of the markings defined in the legend aren't used very often. Is there a common subset that could be displayed by default perhaps? Also, it lookes like there are some items with missing symbols under "memory legend" are those necessary to have?

Reply:
@kenzieschmoll , what you think about removing items without symbols? I did not notice them till Mindy's comment. Sonds as a good option for me.

Reply:
These aren't missing a symbol intentionally. There must be an error in how the legend is be rendered, which we should fix. Every item that is broken has the parameter 'dashed: true'.